automation/run-tests.sh

```diff
@@ -21,40 +18,38 @@ TEST_TYPE_INTEG="integ"
 FEDORA_IMAGE_DEV="nmstate/fedora-nmstate-dev"
 CENTOS_IMAGE_DEV="nmstate/centos8-nmstate-dev"
 
+: ${CONTAINER_CMD:=docker}
+
 test -t 1 && USE_TTY="-t"
 
 function remove_container {
     res=$?
     [ "$res" -ne 0 ] && echo "*** ERROR: $res"
-    docker_exec 'rm -rf $CONTAINER_WORKSPACE/*nmstate*.rpm'
-    docker rm $CONTAINER_ID -f
-    docker network rm $NET0
-    docker network rm $NET1
+    container_exec 'rm -rf $CONTAINER_WORKSPACE/*nmstate*.rpm'
+    ${CONTAINER_CMD} rm $CONTAINER_ID -f
 }
 
 function pyclean {
         find . -type f -name "*.py[co]" -delete
         find . -type d -name "__pycache__" -delete
 }
 
-function docker_exec {
-    docker exec $USE_TTY -i $CONTAINER_ID \
+function container_exec {
+    ${CONTAINER_CMD} exec $USE_TTY -i $CONTAINER_ID \
         /bin/bash -c "cd $CONTAINER_WORKSPACE && $1"
 }
 
 function add_extra_networks {
-    docker network create $NET0 || true
-    docker network create $NET1 || true
-    docker network connect $NET0 $CONTAINER_ID
-    docker network connect $NET1 $CONTAINER_ID
-    docker_exec '
+    container_exec '
+      ip link add eth1 type veth peer eth1peer && \
```

Comment:
Don’t you need to set ‘link up’ on them? At least for the peer ones which we do not suppose to manage using nm.

Reply:
Yeah. `ip link up` is required.

Reply:
Changed it for the peer. It is passing with NM master in the CI, not sure why it fails for 1.20.